### ISSUE TITLE
CSS Cutomization

### ISSUE THREAD

**Issue by cravindra** [joashp/material-design-template#9]
Hey!

Great template thanks for sharing!

Is it possible to include SASS/LESS Files to allow theme customization?

Also, can you please share what your build method is? How did you minify your css and js into the min/.. directory?

**Comment by joashp** [comment]
@cravindra 
Thanks.
Yes, you can include your SASS/LESS files to modify the CSS.
For minification, I use gruntJS. You could also use  https://github.com/mrclay/minify

**Comment by cravindra** [comment]
Hi.
Thanks for the response. I have already tried using my own SASS files and
converted the color codes to variables.. However I just did it
superficially (replaced all instances of the background color with a custom
variable)

I meant to ask if you used any CSS preprocessor to make the theme that we
could use to customize it (the colors, mainly) :)

Thanks for the minification tip. I was coming from the fact that after
recompiling the CSS, I wasn't sure how all the JS and CSS could be minified
automatically into the folder currently in the min folder.
